google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -208,27 +275,42 @@ public String etag() {
    * Sets the version of the policy. The default version is 0, meaning only the "owner", "editor",
```

Comment:
I think this is supposed to stay "sets" since it's the builder's setter method, right?

Just FYI (no changes necessary), according to my (quick/limited) testing this morning, it seems like setting this (at least in cloud resource manager) has no effect.  I think they changed it recently to be read-only.  Unsure about IAM policies in general though.